lib/org/chromium/webidl/URLExtractor.js

```diff
@@ -27,6 +25,7 @@ foam.CLASS({
           });
           return new RegExp(exps.join('|'));
         }
+        return null;
```

Comment:
Line 25: Do we expect to match some FOAMy array-like thing with `foam.Array.isInstance()`? If not, we can just use `Array.isArray()`.